src/openrct2/world/park.c

```diff
@@ -477,6 +477,22 @@ static sint32 park_calculate_guest_generation_probability()
 	return probability;
 }
 
+void get_nearest_peep_spawn(rct2_peep_spawn *spawn, sint32 x, sint32 y)
+{
+	sint32 nearestDist = INT32_MAX;
+	sint32 nearestIndex = MAX_PEEP_SPAWNS;
+	for (sint32 i = 0; i < MAX_PEEP_SPAWNS; i++) {
+		sint32 dx = gPeepSpawns[i].x - x;
+		sint32 dy = gPeepSpawns[i].y - y;
+		sint32 dist = dx * dx + dy * dy;
```

Comment:
@IntelOrca  alright, but before you do merge this please let me know when it's alright to revert the MAX_PEEP_SPAWNS back to 2, as that was only there for allow quicker testing (such as with the builds by travis or appveyor, instead of needing to fetch the PR and compiling yourself, etc)